README.md

```diff
@@ -11,3 +11,12 @@ _**Prerequisits: This repository requires node v16 and up**_
 1. Clone this repository
 2. Install dependecies with `npm i`
 3. Test and lint with `npm test`
+
+## Manual Deployment
+
+Before making a deploy make sure you make a build locally of the content that should be deployed by running `npm run buildPreview`.
+
+Afterwards, run `npm run netlify deploy` to run manual deployments to a unique address that will be generated at the time of deploy. 
```

Comment:
It’s asking me for a publish directory. What’s the right option – `dist` or `.`?

Reply:
dist.

Reply:
do you think I should add that information to the docs??? i guess that can also be inferred from looking at what buildPreview does, but I can also make it clearer here.

Reply:
I definitely think it should be added to the docs, yes.

Reply:
i added it.

Reply:
Hm, I’m not sure that would answer my question… I meant adding it to the paragraph about `npm run netlify deploy`, since that is the command asking about the publish directory.

Reply:
Maybe something like:

> Afterwards, run `npm run netlify deploy` and tell it to publish the `dist` directory, to manually deploy the build to a unique address that will be generated at the time of deploy.

Reply:
I thought this can be inferred from the previous paragraph. But if it's clearer for you adding it as well to this paragraph I can add it.

Reply:
I’d add it here and remove it from the previous paragraph to avoid duplication :) but I think it helps if it’s closer to the deploy command.

Reply:
i think it's good to let the users know where the bundle is being built. this is why i thought it's a better idea to put it in the build step.

Reply:
maybe we can ask a third person what they think? :)

Reply:
I think I see the issue. I wonder if we should go by the old principle of "show, don't tell" here. What if we add a script that actually does all the steps for the developer, and then just instruct them to `npm run deployCommit` or something similar?

I think we can use both `buildPreview` and `netlify deploy --dir="./dist"` for that respect